src/components/10-atoms/radio/ui.test.js

```diff
@@ -81,3 +81,22 @@ test('should show controlled-component behaviour in default mode', async t => {
 
   await t.expect(radio1.getAttribute('checked')).eql('');
 });
+
+fixture('Radio - no label').page(
+  `${host}/iframe.html?id=atoms-radio-demos--radio-no-label`
```

Comment:
The test-url with knobs can be like this example:

  `${host}/iframe.html?id=molecules-datepicker--datepicker&knob-inputfield=true&knob-locale=de-CH&knob-year=2020&knob-month=4&knob-day=22&knob-label=&knob-labelbuttoncancel=Cancel&knob-labelbuttonok=Ok&knob-monthtitle=Choose%20Month&knob-yeartitle=Choose%20Year&knob-invaliddatetext=Invalid%20date&knob-placeholder=Please%20select%20a%20date&knob-width=227&knob-height=80&knob-width%20(note%20max-length)=200`

Reply:
As discussed with raphi in the daily, it is not possible in this case. Therefore I will close this conversation.